core/src/main/resources/lib/layout/html.jelly

```diff
@@ -37,8 +37,7 @@ THE SOFTWARE.
       Title of the HTML page. Rendered into &lt;title> tag.
     </st:attribute>
     <st:attribute name="norefresh">
-      If non-null and not "false", auto refresh is disabled on this page.
-      This is necessary for pages that include forms.
+      Deprecated: Used to disable auto refresh for a page. The feature has been removed.
```

Comment:
Are you deliberately not removing `norefresh` from the many places it is specified, such as `CLIAction/command.jelly` for example?